console/HAL-1595_Configure_support_for_TLS_13.adoc

```diff
@@ -0,0 +1,61 @@
+= Configure support for TLS 1.3
+:author:            Harald Pehl
+:email:             hpehl@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.redhat.com/browse
+
+== Overview
+
+Add supprt for attribute `cipher-suite-names` of resource `/subsystem=elytron/server-ssl-context=*`.
```

Comment:
@hpehl Support for the `cipher-suite-names` attribute should also be added to the `/subsystem=elytron/client-ssl-context=*` resource.

Reply:
Thanks, @fjuma, for the hint! I've updated the analysis document.

Reply:
Yeah, of course. Will do that just now.